teuthology/scrape.py

```diff
@@ -0,0 +1,518 @@
+#!/usr/bin/env python
+
```

Comment:
since we're importing this from another repo, let's add a comment here about its origin and original author. Then add your additional changes to it as another commit, so it's easy to tell what you added